frontend/packages/operator-lifecycle-manager/src/utils.ts

```diff
@@ -0,0 +1,23 @@
+import * as _ from 'lodash';
+import { Annotation, ClusterServiceVersionKind } from './types';
+
+export const getInternalObjects = (annotations: Annotation): string[] => {
+  const INTERNAL = 'operators.operatorframework.io/internal-objects';
+  let internals: string = _.get(annotations, INTERNAL);
+  if (internals) {
+    internals = internals.replace(/'/g, '"');
```

Comment:
Wait, why is this needed? It should parse as JSON with no changes.

Reply:
Ohh sorry, I missed you comment. I have the same concern.

https://github.com/openshift/console/pull/3877#discussion_r366166330

Reply:
ack. I had once gotten a malformed string. Hence put it. But I will remove it.